task-list/app/views/people/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>People</h1>
+<% @people.each do |person| %>
+  <p><%= link_to person.name, "/people/#{person.id}" %> is assigned
+  <% uncomplete = 0 %>
+  <% person.tasks.each do |task| %>
```

Comment:
Is there another way this loop code could be written using `filter` and `count` ?